src/coffeObject.js

```diff
@@ -0,0 +1,246 @@
+const coffeeObject = [
+	{
+		name: "Aynu",
+		profile: "clean", // or "rich",
+		characteristic: "sweet", // or "acidic" or "round"
+		taste: ["chocolate", "spicy", "floral"],
+		processed: "Washed",
+		qGrade: 88,
+		origin: [
+			{
+				name: "Limu",
+				latitude: 9.1667,
+				longitude: 36.34,
+				altitudeMin: 1650,
+				altitudeMax: 1950,
+			},
+			{
+				name: "Limu Kosa",
+				latitude: 8.16638,
+				longitude: 37.17096,
+				altitudeMin: 1750,
+				altitudeMax: 1950,
+			},
+		],
+		name: "Beza",
```

Comment:
It looks like you should have closed the previous object here and started a new one

Reply:
I didn't get what you mean, when I look at it, it seems I closed it at the right place

Reply:
```js
{
  name: "Aynu",
  profile: "clean", // or "rich",
  characteristic: "sweet", // or "acidic" or "round"
  taste: ["chocolate", "spicy", "floral"],
  processed: "Washed",
  qGrade: 88,
  origin: [
    {
      name: "Limu",
      latitude: 9.1667,
      longitude: 36.34,
      altitudeMin: 1650,
      altitudeMax: 1950,
    },
    {
      name: "Limu Kosa",
      latitude: 8.16638,
      longitude: 37.17096,
      altitudeMin: 1750,
      altitudeMax: 1950,
    },
  ],
  name: "Beza",
  profile: "clean", // or "rich",
  characteristic: "sweet", // or "acidic" or "round"
  taste: ["chocolate", "spicy", "floral"],
  processed: "sundried",
  qGrade: 92,
  origin: [
    {
      name: "Yirgachefe",
      latitude: 6.16272,
      longitude: 38.20583,
      altitudeMin: 1900,
      altitudeMax: 2100,
    },
    {
      name: "Yirga Chefe",
      latitude: 6.334640683301095,
      longitude: 39.00524849697445,
      altitudeMin: 1700,
      altitudeMax: 1950,
    },
  ]
  // ...
}
```

vs.

```js
{
  name: "Aynu",
  profile: "clean", // or "rich",
  characteristic: "sweet", // or "acidic" or "round"
  taste: ["chocolate", "spicy", "floral"],
  processed: "Washed",
  qGrade: 88,
  origin: [
    {
      name: "Limu",
      latitude: 9.1667,
      longitude: 36.34,
      altitudeMin: 1650,
      altitudeMax: 1950,
    },
    {
      name: "Limu Kosa",
      latitude: 8.16638,
      longitude: 37.17096,
      altitudeMin: 1750,
      altitudeMax: 1950,
    },
  ],
}, /// <--- Close previous object here
{  /// <--- Open new object here
  name: "Beza",
  profile: "clean", // or "rich",
  characteristic: "sweet", // or "acidic" or "round"
  taste: ["chocolate", "spicy", "floral"],
  processed: "sundried",
  qGrade: 92,
  origin: [
    {
      name: "Yirgachefe",
      latitude: 6.16272,
      longitude: 38.20583,
      altitudeMin: 1900,
      altitudeMax: 2100,
    },
    {
      name: "Yirga Chefe",
      latitude: 6.334640683301095,
      longitude: 39.00524849697445,
      altitudeMin: 1700,
      altitudeMax: 1950,
    },
  ]
  // ...
}
```